forge-gui/res/cardsfolder/m/mark_for_death.txt

```diff
@@ -1,9 +1,10 @@
 Name:Mark for Death
 ManaCost:3 R
 Types:Sorcery
-A:SP$ Pump | Cost$ 3 R | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select target creature an opponent controls | KW$ HIDDEN CARDNAME blocks each combat if able. | IsCurse$ True | SubAbility$ DBUntap | SpellDescription$ Target creature an opponent controls blocks this turn if able. Untap that creature. Other creatures that player controls can't block this turn.
-SVar:DBUntap:DB$ Untap | Defined$ Targeted | SubAbility$ DBEffect
-SVar:DBEffect:DB$ Effect | StaticAbilities$ KWPump | AILogic$ Evasion | RememberObjects$ TargetedController | ImprintCards$ Targeted
-SVar:KWPump:Mode$ Continuous | EffectZone$ Command | AffectedZone$ Battlefield | Affected$ Creature.IsNotImprinted+RememberedPlayerCtrl | AddHiddenKeyword$ CARDNAME can't block. | Description$ Other creatures that player controls can't block this turn.
+A:SP$ Effect | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select target creature an opponent controls | IsCurse$ True | RememberObjects$ Targeted,TargetedController | ForgetOnMoved$ Battlefield | StaticAbilities$ MustBlock,KWPump | SubAbility$ DBUntap | StackDescription$ {c:Targeted} blocks this turn if able. | SpellDescription$ Target creature an opponent controls blocks this turn if able.
```

Comment:
ForgetOnMoved might mess things up here (even if it dies the other part of the effect needs to stay)